pkgs/development/libraries/libjxl/default.nix

```diff
@@ -8,15 +8,21 @@
 , libjpeg
 , libpng
 , libwebp
+, gdk-pixbuf
 , openexr_3
 , pkg-config
+, makeWrapper
 , zlib
 , asciidoc
 , graphviz
 , doxygen
 , python3
 }:
 
+let
+  loadersPath = "${gdk-pixbuf.binaryDir}/jxl-loaders.cache";
```

Comment:
Maybe also include the version number to make chance of conflict even lower but this looks good to me.